src/app_sensors.c

```diff
@@ -42,7 +42,7 @@ static void async_error_handler(struct golioth_client *client,
 
 /* This will be called by the main() loop */
 /* Do all of your work here! */
-void app_work_sensor_read(void)
+void app_sensors_sensor_read(void)
```

Comment:
Since this function does both sensor read and stream to Golioth, do you think it's worth naming it something that reflects it does both? e.g. `app_sensors_read_and_stream` or `app_sensors_stream_sensor_data` etc?

Reply:
Good suggestion, I've updated that function to use `app_sensors_read_and_stream()`